express-js/express.md

```diff
@@ -4,8 +4,8 @@
 4 hours, including lab exercise
 
 ### Prerequisites
-- How the Internet Works
-- Node.js
+- [How the Internet Works](https://github.com/Techtonica/curriculum/blob/master/requests-and-responses/requests-and-responses.md)
```

Comment:
Please use relative paths. Example, replace:
`https://github.com/Techtonica/curriculum/blob/master/html/html.md`
With:
`../html/html.md`

Reply:
If I change the path like `..requests-and-responses/requests-and-responses.md` than it is showing _page not found error_. I am confused

Reply:
Lemme quickly check...

Reply:
@rudri182 It's not `..reques...` it's `../reques...`. The `../` is going to a parent directory or path... 😇

Reply:
👍